modules/ctrl_tcp/ctrl_tcp.c

```diff
@@ -275,7 +275,10 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 	err = odict_entry_add(od, "event", ODICT_BOOL, true);
 	err |= event_encode_dict(od, ua, ev, call, prm);
 	if (err)
+	{
```

Comment:
please move curly bracket to previous line, like this:

```c
if (err) {
}
```

Reply:
done